tokio/src/task/join_set.rs

```diff
@@ -120,6 +120,10 @@ impl<T: 'static> JoinSet<T> {
     /// Spawn the provided task on the `JoinSet`, returning an [`AbortHandle`]
     /// that can be used to remotely cancel the task.
     ///
+    /// You do not have to `.await` the returned `JoinHandle` to make the
```

Comment:
Probably this comment shouldn't be applied to any of the `spawn` methods of `JoinSet`'s. They return `AbortHandle`s.

Reply:
Maybe some form of the comment is appropriate. Something like you do not have to .await the join_next result for the task to begin.